### PR TITLE
1.7.0

### PR DESCRIPTION
**Pull request by orthagh**

### CODE REVIEW

File: inc/container.class.php [hunk 1]
```diff
@@ -1408,4 +1451,12 @@ private static function getSubtypes($item) {
 
       return $tabs;
    }
+
+   static function getClassname($itemtype = "", $raw_name = "") {
```
Comment (trasher) [inline inc/container.class.php]: Missign phpdoc

File: inc/container.class.php [hunk 2]
```diff
@@ -1408,4 +1451,12 @@ private static function getSubtypes($item) {
...
+      return "PluginFields".ucfirst(self::getSystemName($itemtype, $raw_name));
+   }
+
+   static function getSystemName($itemtype = "", $raw_name = "") {
```
Comment (trasher) [inline inc/container.class.php]: Missing phpdoc

File: inc/field.class.php [hunk 3]
```diff
@@ -348,10 +341,10 @@ function showSummary($container) {
 
                echo '<td class="rowhandler control center">';
                echo '<div class="drag row" style="cursor:move;border:none !important;">';
-               echo '<img src="../pics/drag.png" alt="#" title="' . __('Move') .'" width="16" height="16" />';
+               echo '<img src="../pics/drag.png" alt="#" title="'.__('Move').'" width="16" height="16">';
```
Comment (trasher) [inline inc/field.class.php]: Standalone markup is generally closed ;)

Comment (orthagh) [reply inc/field.class.php]: in html5, it's optional

Comment (trasher) [reply inc/field.class.php]: Yes it is... But it is also considered as a good practice to close all tags. Well this is not important.

**Comment by trasher** [comment]
Sounds OK, I've to test the numerous possibilities...

I'm not sure removing the OS stuff is the correct way to go; even if this is a quite huge change :/

**Pull request merged**